10/10-2/exercise4-1.js

```diff
@@ -0,0 +1,26 @@
+const messageDelay = () => Math.floor(Math.random() * 5000);
+
+const getMarsTemperature = () => {
+  const maxTemperature = 58;
+  return Math.floor(Math.random() * maxTemperature);
+};
+
+const toFahrenheit = (degreeCelsius) => (degreeCelsius * 9/5) + 32;
+const temperatureInFahrenheit = (temperature) => console.log(`It is currently ${toFahrenheit(temperature)}ºF at Mars`);
+const greet = (temperature) => console.log(`Hi there! Curiosity here. Right now is ${temperature}ºC at Mars`);
+
+// definição da função sendMarsTemperature...
+
+        
+    const sendMarsTemperature = (callback) => {
+        setTimeout(() =>{
+        const temp = getMarsTemperature();
+        console.log(temp);
+         callback(temp);
+    }, 5100);
+};
```

Comment:
Bacana Mathieu!! Talvez seja melhor usar a função messageDelay no lugar dos 5100 para o setTimeOut
```suggestion
    const sendMarsTemperature = (callback) => {
        setTimeout(() =>{
        const temp = getMarsTemperature();
        console.log(temp);
         callback(temp);
    }, messageDelay());
};
```